command/operator/raft/listpeers/operator_raft_list.go

```diff
@@ -82,8 +103,22 @@ func raftListPeers(client *api.Client, stale bool) (string, error) {
 		if s.Leader {
 			state = "leader"
 		}
-		result = append(result, fmt.Sprintf("%s\x1f%s\x1f%s\x1f%s\x1f%v\x1f%s",
-			s.Node, s.ID, s.Address, state, s.Voter, raftProtocol))
+
+		serverHealthData, ok := serverHealthDataMap[s.ID]
+		if ok {
+			trailsLeaderBy := leaderLastCommitIndex - serverHealthData.LastIndex
```

Comment:
@analogue - Will leader last commit index be always greater than follower? or should I take abs(leaderLastCommitIndex - serverHealthData.LastIndex)